src/components/Avatar/Avatar.module.scss

```diff
@@ -0,0 +1,180 @@
+@import '../../styles/colors';
+
+.Avatar {
+  &:focus-within {
+    --zui-avatar-border-radius: 50%;
+  }
+
+  .AvatarRoot {
+    --zui-avatar-border-radius: 50%;
+    --zui-avatar-width: 2.5rem;
+    --zui-avatar-height: 2.5rem;
+  }
+
+  &[data-type='square'] {
+    .AvatarRoot {
+      --zui-avatar-border-radius: 0.5rem;
+    }
+
+    &:focus-within {
+      --zui-avatar-border-radius: 0.5rem;
+    }
```

Comment:
You should be able to declare these CSS variables outside of the classes, so just in the top-level class.

```
.Avatar {
    &[data-type='square'] {
        --zui-avatar-border-radius: 0.5rem;
    }
}
```

Same goes for most of the others, except for the status ones! Unless you can think of a benefit to keeping them scoped? I may be missing something.

Reply:
I updated the fallback-related styles to the root level. But the other styles are not effecting if we move it to top-level class.